src/ccl_massfunc.c

```diff
@@ -183,18 +183,36 @@ TASK: Outputs fitting function for use in halo mass function calculation;
     ccl_tinker10 (arxiv 1001.3162 )
     ccl_angulo (arxiv 1203.3216 ) 
     ccl_watson (arxiv 1212.0095 )
+    ccl_shethtormen (arxiv 9901122)
 */
 
 static double massfunc_f(ccl_cosmology *cosmo, double halomass, double a, double odelta, int *status)
 {
-  double fit_A, fit_a, fit_b, fit_c, fit_d, overdensity_delta;
+  double fit_A, fit_a, fit_b, fit_c, fit_d, fit_p, overdensity_delta;
   double Omega_m_a;
   double delta_c_Tinker, nu;
   
   double sigma=ccl_sigmaM(cosmo, halomass, a, status);
   int gslstatus;
   
   switch(cosmo->config.mass_function_method) {
+
+  //Equation (10) in arxiv: 9901122
+  // Note that Sheth & Tormen use nu=(dc/sigma)^2 whereas we use nu=dc/sigma
+  case ccl_shethtormen:
+
+    //TODO: Mead: Somehow enforce that this should only work with a virial-density Delta_v
+
+    //ST mass function fitting parameters
+    fit_A = 0.21616;
```

Comment:
I also wonder if these should be defined in the header instead of hard-coded.

Reply:
I feel that maybe these ones shouldn't be in the header since they are taken from papers and the user shouldn't have any choice about what they are. They are on the same level as the fit_A, fit_B, ... that are in the Tinker mass function already.